neo-cli/CLI/Helper.cs

```diff
@@ -19,5 +29,78 @@ public static bool IsYes(this string input)
 
             return input == "yes" || input == "y";
         }
+        public static JObject ToJson(this StackItem item)
+        {
+            return ToJson(item, null);
+        }
+
+        private static JObject ToJson(StackItem item, List<(StackItem, JObject)> context = null)
```

Comment:
```suggestion
        private static JObject ToJson(StackItem item)
```

We only need this method.